blaze/expr/strings.py

```diff
@@ -54,7 +55,14 @@ def isstring(ds):
     return isinstance(getattr(measure, 'ty', measure), String)
 
 
+_add, _radd = _mkbin('add', Concat)
+_mod, _rmod = _mkbin('mod', Interp)
+_mul, _rmul = _mkbin('mul', Repeat)
+
+
 schema_method_list.extend([
+    (isstring, set([_add, _radd, _mod, _rmod, _mul, _rmul,
```

Comment:
@cowlicks I am adding them here because they act on strings, the issue is that that the dtype of a pandas series of strings is just 'object', not 'string'